### PR TITLE
Column Style Update & Project Team View Status Selector & Column Toggle

### PR DESCRIPTION
**Pull request by Cireimu**
# Description
Updates style of most of our Program Manager view related stuff, mainly modals and drop-downs, also adds the status label drop-down selector to the team view level, which could be incorporated with roles and permissions by future teams if need be. Along with both of these, we have incorporated status column toggles from the project list view.

Corresponding back-end request: https://github.com/Lambda-School-Labs/mission-control-be/pull/126

Fixes # (issue)

## Type of change
Feature and tweaks

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

**Review by abdirahmanfarah — APPROVED** [review]

**Comment by Trip4077** [comment]
Everything is running good, after the previous merge you will need to do a pull from the master branch to resolve the conflicts. After that, I think this is good to go

**Review by Trip4077 — APPROVED** [review]

File: src/components/Project/ProjectStatus/index.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  const { data } = state;
+
+  return (
+    <div className={projectStatusDropdown}>
+      {data && data.project.projectStatus.length > 0
+        ? data.project.projectStatus.map(statuses => {
+            return (
+              <div className={dropdownLabel} key={statuses.id}>
+                <h3>{statuses.name}</h3>
+                {/* <ProjectStatusLabel
```
Comment (Memitaru) [inline src/components/Project/ProjectStatus/index.jsx]: If this code is staying here for a reason please add a comment

**Review by Memitaru — APPROVED** [review]

File: src/components/Project/ProjectStatus/index.jsx (hunk 1 above)
Comment (lawsonarichard) [reply src/components/Project/ProjectStatus/index.jsx]: See above^

**Review by lawsonarichard — COMMENTED** [review]
Pulled down with matching [BE branch](https://github.com/Lambda-School-Labs/mission-control-be/pull/126)

On creation of column, I can create a column and received this error
Screen Shot 2020-03-25 at 11 08 37 AM

Screen Shot 2020-03-25 at 11 16 05 AM

After creation of the column I saved and it rendered and persisted. When I went to remove the column the system would hang and freeze. The modal will not allow me to toggle until a hard refresh. 

OS: Mac OS

**Pull request merged**